fuzz/README.md

```diff
@@ -56,6 +55,16 @@ variable).
    -instr-profile=fuzz_packfile_raw.profdata [source file]` shows a line-by-line
    coverage analysis of all the codebase (or a single source file).
 
+## Standalone mode
+
+In order to ensure that there are no regresions, each fuzzer target can be run
+in a standalone mode. This can be done by passing `-DUSE_STANDALONE_FUZZERS=ON`
+to `cmake`. This makes it compatible with gcc. This does not use the fuzzing
+engine, but just invokes every file in the chosen corpus.
+
+In order to get full coverage, though, you might want to also enable one of the
+sanitizers. You might need a recent version of clang to get full support.
+
```

Comment:
That's definitly nice to have. I'm currently stuck on a system without working fuzzers due to problems with musl libc, so I always have to use a VM. At least I can get _some_ feedback with the standalone mode.